conn_test.go

```diff
@@ -190,6 +192,24 @@ func (mock *ftpMock) listen(t *testing.T) {
 			mock.proto.Writer.PrintfLine("200 NOOP ok.")
 		case "REIN":
 			mock.proto.Writer.PrintfLine("220 Logged out")
+		case "SITE":
+			if cmdParts[1] == "QUOTA" {
+				response := []string{}
```

Comment:
Please initialize the response array with all its values and avoid `append`.
```golang
response := []string{
  "firstline",
  "secondline",
  "...",
}
```

Reply:
This was also updated as recommended.